packages/react-instantsearch-dom-maps/src/Control.js

```diff
@@ -69,4 +62,4 @@ export class Control extends Component {
 export default translatable({
   control: 'Search as I move the map',
   redo: 'Redo search here',
-})(Control);
+})(withGoogleMaps(Control));
```

Comment:
complex bracketing here, but is expected I guess. Don't we usually write this in the other order?

Reply:
The order doesn't matter, there is no dependency between the two HoCs. The proper way to write this would be with a `compose` function.

```js
export default compose(
  withGoogleMaps,
  translatable({ ... })
)(Control)
```

Reply:
the confusion was that `withGoogleMaps`. isn't `() => () => () =>`, but `() => () =>`, so it doesn't need its own invocation for options.